test/jobs/verify_linkbacks_job_test.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class VerifyLinkbacksJobTest < ActiveJob::TestCase
+  LINKBACK_HTML = <<~HTML
+    <html>
+      <head>
+        <title>Site with valid linkbacks</title>
+        <link rel="rubygem" href="https://rubygems.org/gem/mygem">
```

Comment:
This is a general question, but is the HTML required to have both the `<link >` and the `<a>` with the correct `rel` attr to be valid? Or just one of them? The examples in the issue and in here have both but the way that the code is written, you only need one. Just trying to understand :)

Reply:
It's either one or the other — <link> in the head or <a> in the HTML body. I just stuck 'em both into the sample. :D